### DISCUSSION TITLE
请问作者可以添加一下这9x阅读器吗

### DISCUSSION THREAD

**Discussion by 13316282905** [xiaohucode/yidaRule#104] [Ideas]
![Screenshot_2024-03-11-18-14-09-715_com miui home](https://github.com/xiaohucode/yidaRule/assets/159050612/a10a179d-01ff-426e-a3bc-6b4464086827)
我不会添加，你帮我加一下可以吗

**Comment by 13316282905** [comment]
请作者添加一下9x阅读器规则进这软件

**Comment by 13316282905** [comment]
请作者帮我添加一下这9x阅读器的漫画规则。我不会弄

**Comment by xiaohucode** [comment]
不能
